ignite/templates/typed/options.go

```diff
@@ -24,3 +30,13 @@ type Options struct {
 func (opts *Options) Validate() error {
 	return nil
 }
+
+// Path to proto folder in the generated app.
+func (opts *Options) ProtoPath(fname string) string {
+	return filepath.Join(opts.AppPath, "proto", opts.AppName, opts.ModuleName, fname)
+}
+
+// Return the import statement for this type.
+func (opts *Options) TypeImport() *proto.Import {
```

Comment:
Does it make sense to use "proto" in the function name to make it more specific?

```suggestion
// ProtoImport returns the proto import statement for this type.
func (opts *Options) ProtoImport() *proto.Import {
```

Or `ProtoTypeImport`.

Reply:
I think I initially used `ProtoImport` but that seemed vague since we're talking about a specific type. I opted for the second suggestion here, `ProtoTypeImport`, thanks!